website/docs/cdktf/concepts/modules.mdx

```diff
@@ -93,6 +93,12 @@ For performance reasons, we don't automatically generate bindings for submodules
 
 Go to the working directory and run `cdktf get`. CDKTF automatically creates the appropriate module bindings in the `./.gen` directory for you to use in your application.
 
+### Configuring Modules
```

Comment:
I think we might want to have this as an H2 and call it "Configure Modules" to match the construction we use elsewhere :-)  This isn't a step to Install Modules, right? That's why I think we make a new H2